Dockerfile.dapper

```diff
@@ -62,19 +59,59 @@ RUN mkdir /usr/tmp && \
     mv /usr/tmp/helm /usr/bin/helm_v3 && \
     chmod +x /usr/bin/kustomize
 
-RUN curl -sLf ${!K3S_DOWNLOAD_URL} -o /usr/bin/k3s && \
-    chmod +x /usr/bin/k3s && \
-    mkdir -p /go/src/github.com/rancher/rancher/.kube && \
+# Set up K3s: copy the necessary binaries from an archive with extracted version of K3s.
+COPY --from=rancher/k3s:v1.22.3-k3s1 \
+    /bin/blkid \
+    /bin/charon \
+    /bin/check-config \
+    /bin/cni \
+    /bin/conntrack \
+    /bin/containerd \
+    /bin/containerd-shim-runc-v2 \
+    /bin/ethtool \
+    /bin/fuse-overlayfs \
```

Comment:
you can drop this; it's only needed for rootless

Reply:
@brandond, are these and others above not needed in `package/Dockerfile` as well?

Reply:
Also, remove `containerd`? But k3s binaries (k3s, k3s-agent, etc.) are linked to it...

Reply:
things removed from here can also be removed from package/Dockerfile.

don't remove containerd, just the specific lines that I left comments on:
```
/bin/check-config
/bin/fuse-overlayfs
/bin/slirp4netns
/bin/socat
```